Lib/site.py

```diff
@@ -465,7 +465,7 @@ def venv(known_paths):
 
     if candidate_confs:
         virtual_conf = candidate_confs[0]
-        system_site = "true"
```

Comment:
Any reason we use a string here instead of a bool?

Reply:
It's parsed by C code during startup, so parsing facilities are limited.